lib/views/mount-row.js

```diff
@@ -2,36 +2,32 @@
 
 import React from 'react'
 
+import { MachineStatus } from '../constants'
+
 const MountRow = ({ mount, index, onOpen, onNewWindow, onUnmount }) => {
   const _onOpen = () => onOpen(mount, index)
   const _onNewWindow = () => onNewWindow(mount, index)
   const _onUnmount = () => onUnmount(mount, index)
 
+  if (isOffline(mount)) {
+    return (
+      <tr>
+        <td>{mount.label}</td>
+        <td colSpan='4'>
+          Mounted machine is offline.
+          Close this window and run <strong>kd:machines</strong> to turn it on.
```

Comment:
we can either do:
- `Go to <machine settings> to turn it on.` where machine settings is a link to kd:machines
or
- `<Repair this mount> by turning the machine on and remounting to the same folder` where 'Repair this mount' turns the machine on, unmounts and remounts.